lib/rspec/rails/example/system_example_group.rb

```diff
@@ -38,10 +38,18 @@ def passed?
 
       # @private
       def method_name
-        @method_name ||= [
-          self.class.name.underscore,
-          RSpec.current_example.description.underscore
-        ].join("_").tr(CHARS_TO_TRANSLATE.join, "_")[0...200] + "_#{rand(1000)}"
+        @method_name ||= begin
+          m = [
+            self.class.name.underscore,
+            RSpec.current_example.description.underscore
+          ].join("_").tr(CHARS_TO_TRANSLATE.join, "_")
+
+          if RbConfig::CONFIG["host_os"] =~ /linux/
```

Comment:
I thought we had something for that in https://github.com/rspec/rspec-support/blob/12069e1c1ad1852b7970e7fcd419d56a8a9e6240/lib/rspec/support/ruby_features.rb#L9 but this is not the case.

Reply:
Don't make it OS specific, it's fine to be shorted on all platforms to satisfy linux.